### DISCUSSION TITLE
Ma Lunii n'est pas reconnue par STUdio

### DISCUSSION THREAD

**Discussion by veetrop** [marian-m12l/studio#340] [Q&A]
j'ai installé le Luniistore, STUdio 0.4.2, et Java 19.0.1. Mon Mac OS est Ventura 13.1 tournant sur un MacBook Pro 15" 2018. Tout fonctionne très bien Mais STUdio s'entête à ne pas reconnaître ma Boîte à Histoires .... Grrrrr!!! Savez-vous qui pourrait m'aiguiller sur une réponse pratique et fonctionnelle? J'ai fait un joli Pack à histoire (grâce à vous !) mais qu'en faire maintenant pour le transférer sur ma Lunii ? Tant de travail conclu par un échec si prêt du but (Merci pour vos super tutos!) D'autant que la boîte est parfaitement reconnue par le Luniistore, donc pas de pb de câble USB. Une bonne âme pour m'aider ? Merci à tous

**Comment by Hermi16** [comment]
avez vous trouvez la solution ? merci

**Comment by benjaminalhadef** [comment]
idem depuis la dernière mise à jour de Ventura. ce problème est lié à celui-ci : https://github.com/marian-m12l/studio/issues/327#issue-1453313622
Du coup, obligé de trouver un Windows pour le faire...

**Comment by veetrop** [comment]
Bonjour Hermi16

Et bien non, pas de solution pour l’instant…
Etes-voues dans le même cas que moi?
Si ça fonctionne chez vous, quelle est votre configuration ?
Bonne journée
Franck
